src/common/components/testimonial/testimonial.css

```diff
@@ -22,25 +22,22 @@
   }
 }
 
+.pullQuote>p>span {
+  color: #dd4814;
+  font-size: 2rem;
+  line-height: 0;
+  position: relative;
+  top: 0.7rem;
+  left: -0.1rem;
+}
+.pullQuote>p>span + span{
+  left: 5px;
+}
 .pullQuote>p {
   font-size:1em;
   font-style:normal;
-  margin:0
-}
-
-.pullQuote>p:first-of-type:before {
```

Comment:
Yep, as far as I remember it was direct copy from vanilla or other pattern at that point.
But we could clean it up a bit when touching.